index.md

```diff
@@ -8,8 +8,8 @@ permalink: /
 <h2 class="marg-b-4">{% t home.summary %}</h2>
 
 <div class="marg-b-3 p-3 secondaryBg">
-  <h3>{% t home.black_lives_matter.title %}</h3>
-  <p>{% t home.black_lives_matter.description %}</p>
+  <h3>{% t home.looking_for_union.title %}</h3>
```

Comment:
Given this is not translated, and it's English language centric to begin with I'd keep it as. And this minimizes the changes to other languages, while keeping this change English only, as that's our goal. Consistency is not as important, as having localized language knowledge available 

{% if site.lang == 'en' %}
  <h3>{% t home.looking_for_union.title %}</h3>
<p>{% t home.looking_for_union.description %}</p>
{%else %}
  <h3>{% t home.black_lives_matter.title %}</h3>
  <p>{% t home.black_lives_matter.description %}</p>
{% endif %}